_posts/2016-12-06-the-state-of-babel.md

```diff
@@ -0,0 +1,532 @@
+from stage 2 to stage 3.
+
+---
+
+Next we're going to be working with:
+
+- [Daniel Ehrenberg](https://github.com/littledan) on [Private Fields (Stage 2)](https://github.com/tc39/proposal-private-fields)
+- [Yehuda Katz](https://github.com/wycats) on [Class and Property Decorators (Stage 2)](http://tc39.github.io/proposal-decorators/)
+- [Mathias Bynens](https://mathiasbynens.be/) on [Unicode Property Escapes in Regular Expressions (Stage 2)](https://github.com/tc39/proposal-regexp-unicode-property-escapes) via [babel/babel#3683](https://github.com/babel/babel/pull/3683)
+
+Relevant Issues:
+
+- Maybe we should think about creating a codemod for Stage x proposals at the same time as creating the actual transform.
+- [Private Fields](https://github.com/babel/babel/issues/4408)
+- [Decorators Proposal](https://github.com/babel/babel/issues/2645)
+
+### Maintaining other ecosystem plugins: JSX/Flow
+
+Babel is vital to the React and Flow ecosystems, and we work closely with the relevant teams at Facebook.
+
+- [React plugins](https://babeljs.io/docs/plugins/#react), [React preset](https://babeljs.io/docs/plugins/preset-react/)
+  - These cover the main `JSX` transform, development plugins, and optimizations.
+- [Flow plugin](https://babeljs.io/docs/plugins/transform-flow-strip-types)
+
+Relevant Issue Labels:
+
+- [babel/babel: React issues](https://github.com/babel/babel/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20label%3A%22react%22%20)
+- [facebook/react: Optimizing Compiler](https://github.com/facebook/react/labels/Component%3A%20Optimizing%20Compiler)
+
+### [babel-preset-env](https://github.com/babel/babel-preset-env) ("autoprefixer" for Babel)
+
+JavaScript compilation is a moving target: There are yearly updates to the spec, browser vendors are constantly updating to that spec, and users may drop support for earlier browsers. At first glance, there doesn't seem to be a fixed target for what we should compile our JavaScript down to.
+
+This is where `babel-preset-env` comes in: it's a Babel preset that automatically determines the correct Babel plugins to use based on the provided environments.
+
+Its goal is both simplicity in use and efficiency in output: you only need to worry about your target environments to be able to take advantage of native code. The preset decides for you the required plugins.
+
+#### Some example configs
+
+Targeting Chrome 55 + last 2 versions of other browsers via [browserslist](https://github.com/ai/browserslist)
+
+```js
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "chrome": 55,
+        "browsers": ["last 2 versions"]
+      }
+    }]
+  ]
+}
+```
+
+Targeting the current Node.js version (uses `process.versions.node`)
+
+```js
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "current"
+      }
+    }]
+  ]
+}
+```
+
+Relevant Issues:
```

Comment:
Nit: "Relevant Issues" is in the middle of the examples section, between examples. Should probably be _above_ or _below_ so that examples are all grouped together.